modules/apps/headless/headless-user-notification/headless-user-notification-test/src/testFunctional/macros/UserNotificationAPI.macro

```diff
@@ -0,0 +1,129 @@
+definition {
+
+	macro _curlNotificationTemplates {
+		var portalURL = JSONCompany.getPortalURL();
+
+		if (isSet(notificationTemplateId)) {
+			var api = "notification-templates/${notificationTemplateId}";
+		}
+		else {
+			var api = "notification-templates";
+		}
+
+		var curl = '''
+			${portalURL}/o/notification/v1.0/${api} \
+				-u test@liferay.com:test \
+				-H accept: application/json \
+				-H 'Content-Type: application/json' \
```

Comment:
Compared to the below _curlUserNotifications(), there is no the following code, and I think application/json is  the default content-type, could you make sure if we can remove it from here? Or add both of them in _curlNotificationTemplates() and _curlUserNotifications().

> -H 'Content-Type: application/json'

Reply:
I have tried locally, the curlNotificationTemplates need the -H 'Content-Type: application/json', may be we can add both of them.

Reply:
Good to know, then we don't need to update here, thanks for checking :+1: